src/main/java/org/killbill/billing/client/model/Tier.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2016 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.client.model;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Tier {
+
+    private final List<TieredBlock> blocks;
+
+    @JsonCreator
+    public Tier(@JsonProperty("tiers") final List<TieredBlock> blocks) {
```

Comment:
Aren't we also missing limits and prices for the [capacity in arrear](https://github.com/killbill/killbill/blob/master/catalog/src/test/resources/UsageExperimental.xml#L170-L193) use-case?